src/compiler/crystal/codegen/target.cr

```diff
@@ -64,8 +67,6 @@ class Crystal::Codegen::Target
   def freebsd_version
     if @environment =~ /freebsd(\d+)\.\d+/
       $1.to_i
-    else
-      nil
```

Comment:
Please remove this unrelated cosmetic change.

Reply:
Is it really necessary?

Reply:
Is it really neccesary to include unrelated changes in PRs?

I write code like this because I think it's clearer and more explicit, not because I missed a trick.

Reply:
Ok, I've reverted this change.